SA3/qa_web/urls.py

```diff
@@ -1,6 +1,11 @@
 from django.urls import path
+from django.conf.urls import url
 from . import views
 
 urlpatterns = [
-    path('', views.index, name='index')
+    path('', views.index, name='index'),
+
+    path(r'login/', views.login, name='login'),
+
```

Comment:
Next time, delete these line breaks; it does not matter to have all the `urls` condensed